docs/content/latest/quick-start/build-apps/nodejs/ycql.md

```diff
@@ -107,7 +107,7 @@ async.series([
     client.execute(select, function (err, result) {
       if (err) return next(err);
       var row = result.first();
-      const city = JSON.parse(row.location).city;
+      const city = row.location.city;
```

Comment:
This change is correct, but we also need line 97 to change, as the INSERT statement is missing `location`:

```js
const insert = "INSERT INTO ybdemo.employee (id, name, age, language, location) " +
```

Reply:
@polarweasel I made the changes as suggested. But it looks to me like I messed up with git. So I'll see if I can fix that.